app/main.py

```diff
@@ -2,24 +2,76 @@
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: "Visitor", name: str) -> None:
+        self.public_name = name
+        self.private_name = "_" + name
+
+    def __get__(self, instance: "Visitor", value: int) -> str:
+        return getattr(instance, self.private_name)
+
+    def __set__(self, instance: "Visitor", value: int) -> None:
+        if value != -1 and not self.min_amount <= value <= self.max_amount:
+            raise ValueError(f"Value {value} is out of range - "
```

Comment:
After opening the parentheses, write string on a new line, including the first one, the closing bracket is also written on a new line